packages/phone-number-privacy/combiner/test/end-to-end/resources.ts

```diff
@@ -33,9 +33,11 @@ export const getTestContextName = (): OdisContextName => {
 export const DEFAULT_FORNO_URL =
   process.env.ODIS_BLOCKCHAIN_PROVIDER ?? 'https://alfajores-forno.celo-testnet.org'
 
-export const PRIVATE_KEY = '2c63bf6d60b16c8afa13e1069dbe92fef337c23855fff8b27732b3e9c6e7efd4'
+export const PRIVATE_KEY = '2c63bf6d60b16c8afa13e1069dbe92fef337c23855fff8b27732b3e9c6e7efd4' // XXX only valid for staging and alfajores
 export const ACCOUNT_ADDRESS = normalizeAddressWith0x(privateKeyToAddress(PRIVATE_KEY)) // 0x6037800e91eaa703e38bad40c01410bbdf0fea7e
 
+// export const PRIVATE_KEY_NO_QUOTA =
+// '2c63bf6d60b16c8afa13e1069dbe92fef337c23855fff8b27732b3e9c6e7efd4' // XXX use this PK on mainnet
```

Comment:
should this comment be here?

Reply:
I can remove, but I figured it might be worth having here for when e2e testing on mainnet. It's also not exposing any new PK,